redbot/cogs/mod/events.py

```diff
@@ -72,6 +71,60 @@ async def check_mention_spam(self, message):
                         print(e)
                         return False
                     return True
+
+        if mention_spam["kick"]:
+            if len(mentions) >= mention_spam["kick"]:
+                try:
+                    await guild.kick(author, reason=_("Mention Spam (Autokick)"))
+                except discord.HTTPException:
+                    log.info(
+                        "Failed to kick member for mention spam in server {}".format(guild.id)
+                    )
+                else:
+                    try:
+                        await modlog.create_case(
+                            self.bot,
+                            guild,
+                            message.created_at,
+                            "kick",
+                            author,
+                            guild.me,
+                            _("Mention spam (Autokick)"),
+                            until=None,
+                            channel=None,
+                        )
+                    except RuntimeError as e:
+                        print(e)
+                        return False
+                    return True
+
+        if mention_spam["warn"]:
+            if len(mentions) >= mention_spam["warn"]:
+                try:
+                    await author.send(
+                        _("Please do not mention {} or more people!".format(mention_spam["warn"]))
+                    )
+                except (discord.HTTPException, discord.Forbidden):
```

Comment:
A fallback option for this could be mentioning the user in the channel the mention spam occurred in, since users could just have "allow direct messages from server members" disabled.

Reply:
Accounted, removed the number telling. Changed to `Please do not mass mention people!`, if the DM fails. It'll send in the channel `@member, Please do not mass mention people!`